cmd/providers.go

```diff
@@ -31,6 +33,7 @@ func init() {
 		"Don't ask for a username and use this instead")
 	cmdProvidersCreateOneLogin.Flags().StringVar(&region, "region", "US",
 		"Region in which the OneLogin API lives")
+	cmdProvidersCreateOneLogin.Flags().IntVar(&providerDuration, "duration", 14400, "(Optional) Default session duration validity assumed with this provider")
```

Comment:
I don't understand the help text. We want to write text that the users will understand easily. Simple is good.

Reply:
rephrased